core/store/LevelDBStore/LevelDBStore.go

```diff
@@ -433,6 +403,26 @@ func (bd *LevelDBStore) GetBlock(hash Uint256) (*Block, error) {
 
 func (bd *LevelDBStore) persist(b *Block) error {
 
+	unspents := make(map[Uint256][]uint16)
+	///////////////////////////////////////////////////////////////
+	// Get Unspents for every tx
+	unspentprefix := []byte{byte(IX_Unspent)}
+	for i:=0;i<len(b.Transactions); i++ {
+		txhash := b.Transactions[i].Hash()
+		unspentvalue, err_get := bd.Get(append(unspentprefix, txhash.ToArray()...))
+
+		if err_get != nil {
+			unspentvalue = []byte{}
+		}
+
+		unspents[txhash],err_get = GetUint16Array(unspentvalue)
+		if err_get != nil {
```

Comment:
Ditto , keep the same style and space unspents[txhash],err_get

to  unspents[txhash], err_get